src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,18 @@
+import { LinkProps } from "@/constants/index.types";
+import Card from "./Card/Card";
+import styles from "./CardList.module.scss";
+
+interface CardListProps {
+  links?: LinkProps[] | undefined;
+}
+
+export default function CardList({ links }: CardListProps) {
+  return (
+    <section className={styles.CardList}>
+      {links &&
+        links.map((link: LinkProps) => {
```

Comment:
타입이 자동으로 추론되기 때문에 별도로 작성하지 않아도 괜찮을 것 같네요!
```suggestion
        links.map((link) => {
```